client/v3/concurrency/session.go

```diff
@@ -116,6 +120,8 @@ func WithTTL(ttl int) SessionOption {
 	return func(so *sessionOptions) {
 		if ttl > 0 {
 			so.ttl = ttl
+		} else {
+			logger.Info("TTL should be > 0, defaulting to 60")
```

Comment:
Should log `defaultSessionTTL` constant not `60`. For example `logger.Info("TTL should be > 0, using default", zap.Int64("default-session-ttl", defaultSessionTTL))`

Reply:
Great catch! Thank you!!